CMakeLists.txt

```diff
@@ -133,13 +133,13 @@ endif()
 eprosima_find_package(fastcdr)
 eprosima_find_package(fastrtps OPTION TINYXML2_FROM_SOURCE)
 
-# Required by ContentFilterTopic
+# Required by ContentFilteredTopic
 if (NOT fastrtps_FOUND OR fastrtps_VERSION LESS 2.5.1)
     message(FATAL_ERROR "Trying to use a version of Fast-DDS lower than 2.5.1. Please update your Fast-DDS installation.")
 endif()
 
 if(BUILD_ROS_COMPONENTS)
-    cmake_policy(SET CMP0057 NEW)
+    # cmake_policy(SET CMP0057 NEW)
```

Comment:
If the policy is not used, I suggest removing it instead of leaving phantom code.